lib/audited/auditor.rb

```diff
@@ -319,17 +310,28 @@ def auditing_enabled=(val)
         Audited.store["#{table_name}_auditing_enabled"] = val
       end
 
-      protected
       def default_ignored_attributes
-        [primary_key, inheritance_column] + Audited.ignored_attributes
+        [primary_key, inheritance_column] | Audited.ignored_attributes
```

Comment:
I changed this to a `|` because if `Audited.ignored_attributes` because it is not inconceivable that `+` could lead to duplicate values say if someone configured `Audited.ignored_attributes` to include `inheritance_column`.